osu.Game/Rulesets/Mods/IApplicableToSampleMixer.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Audio.Mixing;
+
+namespace osu.Game.Rulesets.Mods
+{
+    /// <summary>
+    /// An interface for mods that add effects to the sample mixer.
+    /// </summary>
+    public interface IApplicableToSampleMixer : IApplicableMod
+    {
+        void ApplyToSampleMixer(AudioMixer mixer);
+    }
```

Comment:
> - **Should IApplicableToSampleMixer exist?**
Yes for symmetry, but muted mod probably won't be needing this since it requires separate control of hit samples and metronome samples. Feel free to revert https://github.com/ppy/osu/commit/016e0c320745dae61a9c273d4bbb46d382392baa if this is deemed redundant.

I would say it's best not to provide a mixer for samples currently. I understand it's for symmetry, but it's not clear when a mod will actually want to apply an effect to the global sample mixer, or any specific sample mixers if at all.

Reply:
Reverted the addition of `IApplicableToSampleMixer`